packages/mercurius/lib/interfaces/mercurius-plugin.interface.ts

```diff
@@ -5,6 +5,9 @@ import {
   FastifyRegisterOptions,
 } from 'fastify';
 
+/**
+ *  @publicApi
```

Comment:
```suggestion
 * @publicApi
```
sometimes there are 2 spaces instead of one 😄 not sure if that's even relevant though

Reply:
Done! see commit here: [7978a91](https://github.com/nestjs/graphql/pull/3475/commits/7978a919a892ffd1fc987944c0273c454a038936)